bootstrap/solar-master.json

```diff
@@ -41,9 +86,6 @@
       "playbook_dir": "playbooks",
       "playbook_file": "playbooks/build-main.yaml",
       "extra_arguments": ["--verbose"]
-    }],
-  "post-processors": [{
```

Comment:
we need this provisioner. Add name to vagrant builder and add this option here:
`"only": ["master-vagrant"]`